bin/tests

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
```

Comment:
Is this file sourced in a way where the BASH_SOURCE[0] is needed, or could the simpler

`$( cd "${0%/*}" && pwd )`

be used?

Reply:
This is how `bindir` is created in all the scripts. I'd rather not change it in a single place here since I'm not sure if that would continue to work on all shells. If it does continue to work, that can be a separate change at some point.

Reply:
There is in fact 22 script files in the `bin `dir today where it is not, but done this POSIX way.
In some cases where bash is used and sourced in different ways, the BASH_SOURCE is needed (and there are 25 such script files in the `bin `dir), although I do not think this file is one of those cases.

Reply:
Yep looks like it's a big mix. I'm going to keep this as is for now still. It's not related to this actual change and is how it was before. If it is actually better to change this, I think that can be done everywhere all at once in a single PR.

Reply:
I guess not a big mix, but using Bash functionality where it is needed (in some of those occurrences not? and POSIX where there is no need for extra bells & whistles. Regardless of what I agree that your strategy of handling that separately is good.